From 32eda9ada3dade4c9f6a8a176ee326ccdd61fda8 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 1 Oct 2013 00:06:48 +0200
Subject: [PATCH] main: don't free fds array twice

---
 src/core/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/core/main.c b/src/core/main.c
index 72bd542..662d16d 100644
--- a/src/core/main.c
+++ b/src/core/main.c
@@ -1567,6 +1567,7 @@ int main(int argc, char *argv[]) {
         /* This will close all file descriptors that were opened, but
          * not claimed by any unit. */
         fdset_free(fds);
+        fds = NULL;
 
         if (serialization) {
                 fclose(serialization);
