From 8ace570670d71d13fe604e7cf9d264de245ac981 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Mon, 21 Jul 2014 15:13:15 +0200
Subject: [PATCH] units: order network-online.target after network.target

Resolves: #1121650
---
 units/network-online.target | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/network-online.target b/units/network-online.target
index a40c44c..67bc4fa 100644
--- a/units/network-online.target
+++ b/units/network-online.target
@@ -9,3 +9,4 @@
 Description=Network is Online
 Documentation=man:systemd.special(7)
 Documentation=http://www.freedesktop.org/wiki/Software/systemd/NetworkTarget
+After=network.target
