From 3c66bfddf2038b3d999045dffd1b3beefbb1f8f4 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <fsumsal@redhat.com>
Date: Thu, 17 Jan 2019 13:21:42 +0100
Subject: [PATCH] travis: drop the test_path_changed workaround

---
 ci/travis-centos-rhel7.sh | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/ci/travis-centos-rhel7.sh b/ci/travis-centos-rhel7.sh
index 9f128ab8e2..7db3a8db41 100755
--- a/ci/travis-centos-rhel7.sh
+++ b/ci/travis-centos-rhel7.sh
@@ -52,13 +52,6 @@ for phase in "${PHASES[@]}"; do
                                      --enable-gtk-doc --enable-compat-libs --disable-sysusers \
                                      --disable-ldconfig --enable-lz4 --with-sysvinit-path=/etc/rc.d/init.d
             $DOCKER_EXEC make
-            # Temporarily skip test_path_changed in src/test/test-path.c
-            # This particular test case timeouts on Ubuntu in a Docker image
-            # and needs further debugging (couldn't reproduce it on
-            # Fedora + Docker
-            $DOCKER_EXEC sed -i '/static void test_path_changed/,/^}\s*$/d' src/test/test-path.c
-            $DOCKER_EXEC sed -i '/test_path_changed,/d' src/test/test-path.c
-
             # Run the internal testsuite
             # Let's install the new systemd and "reboot" the container to avoid
             # unexpected fails due to incompatibilities with older systemd
