From 19caf2d160b30878db4e035868747a83087305dd Mon Sep 17 00:00:00 2001
From: Fedora systemd team <systemd-maint@redhat.com>
Date: Thu, 22 Jan 2015 11:24:13 +0000
Subject: [PATCH] CP-9798: Disable journal storage and rate limiting

---
 src/journal/journald.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/journal/journald.conf b/src/journal/journald.conf
index 47eefe9..58728bc 100644
--- a/src/journal/journald.conf
+++ b/src/journal/journald.conf
@@ -12,13 +12,13 @@
 # See journald.conf(5) for details.
 
 [Journal]
-#Storage=auto
+Storage=none
 #Compress=yes
 #Seal=yes
 #SplitMode=uid
 #SyncIntervalSec=5m
-#RateLimitInterval=30s
-#RateLimitBurst=1000
+RateLimitInterval=0s
+RateLimitBurst=0
 #SystemMaxUse=
 #SystemKeepFree=
 #SystemMaxFileSize=
@@ -27,7 +27,7 @@
 #RuntimeMaxFileSize=
 #MaxRetentionSec=
 #MaxFileSec=1month
-#ForwardToSyslog=no
+ForwardToSyslog=yes
 #ForwardToKMsg=no
 #ForwardToConsole=no
 #ForwardToWall=yes
